android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIFloatingActionButton.java

```diff
@@ -54,6 +55,9 @@ public void processProperties(KrollDict d)
 		if (d.containsKey("maxImageSize")) {
 			setMaxImageSize(TiConvert.toInt(d.get("maxImageSize")));
 		}
+		if (d.containsKey(TiC.PROPERTY_TINT_COLOR)) {
+			setTintColor(d.getString("tintColor"));
```

Comment:
Can you add docs for the new property please?

Reply:
ups, sorry. I only looked at the docs and saw it but it was under "excludes" :smile: